bson/src/main/org/bson/BsonNumber.java

```diff
@@ -19,7 +19,7 @@
 import org.bson.types.Decimal128;
 
 /**
- * Base class for the three numeric BSON types.  This class mirrors the functionality provided by {@code java.lang.Number}.
+ * Base class for the numeric BSON types. This class mirrors the functionality provided by {@code java.lang.Number}.
```

Comment:
Actually, there are four numeric BSON types. I removed the number here because we forgot to update it previously (when `BsonDecimal128` was added) and may forget in the future if more types are added.